source/common/http/conn_manager_impl.h

```diff
@@ -160,7 +160,7 @@ enum class ForwardClientCertType {
  * Configuration for the fields of the client cert, used for populating the current client cert
  * information to the next hop.
  */
-enum class ClientCertDetailsType { Subject, SAN };
+enum class ClientCertDetailsType { Subject, SAN, Cert };
```

Comment:
Nit: this should probably be first item, not the last.

Reply:
I assume there's no worry about changing the number values then, or should I be explicit and do { Cert=2, Subject=0, SAN=1 }?

Reply:
Changing the values is fine.